workflow/proc_pipe/fmriprep/scripts/sample_fmriprep_sge.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+#$ -cwd
+#$ -o logs/fmriprep_out.log
+#$ -e logs/fmriprep_err.log
+#$ -m e
+#$ -l h_rt=24:00:00
+#$ -l h_vmem=32G
+#$ -q origami.q
+
+#$ -t 2-11
+
+source /data/origami/nikhil/my_envs/mr_proc_env/bin/activate
+
+SUBJECT_LIST="<>"
```

Comment:
Maybe write somewhere that the `<>` should be filled in with real paths by the user? Or add `# TODO replace` comments whenever there is `<>`